phpstan-baseline.neon

```diff
@@ -1,6 +1,16 @@
 parameters:
-	ignoreErrors:
-		-
-			message: "#^Class Sonata\\\\UserBundle\\\\Model\\\\User not found\\.$#"
-			count: 2
-			path: src/Timezone/UserBasedTimezoneDetector.php
+    ignoreErrors:
+        -
+            message: "#^Class Sonata\\\\UserBundle\\\\Model\\\\User not found\\.$#"
```

Comment:
Can you add a comment to explain why userBundle is not a dev-dependency ? :)

Reply:
Added